test/discrete_test.json

```diff
@@ -1376,6 +1376,33 @@
     }
   ], 
   [
+    "Poisson(0.0)", 
```

Comment:
It is to add a line to `test/discrete_test.lst`. You should see things there as:
```
Poisson()
Poisson(0.5)
Poisson(2.0)
Poisson(10.0)
Poisson(80.0)
```
Then you may add `Poisson(0.0)` over there, and re-run `predref.py` to update the json file.